survey/questions/translation_first_draft.md

```diff
@@ -38,12 +48,14 @@ Male
 
 Female
 
+Other
```

Comment:
I added an option for users who want to declare their gender, but do not identify as male or female !